src/main/webapp/app/shared/links/TextWithRefs.tsx

```diff
@@ -0,0 +1,47 @@
+import { REFERENCE_LINK } from 'app/config/constants/constants';
+import { ParsedRef, parseReferences } from 'app/oncokb-commons/components/RefComponent';
+import React from 'react';
+
+export interface ITexWithRefsProps {
+  content: string;
+}
+
+/* eslint-disable @typescript-eslint/prefer-regexp-exec */
+export default function TextWithRefs({ content }: ITexWithRefsProps) {
+  const result: JSX.Element[] = [];
+  const parts = content?.split(REFERENCE_LINK) || [];
+
+  function formatReferences(refs: ParsedRef[]) {
```

Comment:
Can we move this function out of this component, so the var definition is closer to where it's actually used?